### DISCUSSION TITLE
如何在【原创性说明】和【致谢】中添加一页？（不计入目录）

### DISCUSSION THREAD

**Discussion by chengzhongyi8** [TheNetAdmin/zjuthesis#376] [General]
操作系统：Overleaf
GradLevel：doctor

拿到了以前学长学姐的模板，发现在【原创性说明】和【致谢】之间还有一页是基金的资助信息。如图：
![WeChat6ecbec67a10b5ce5068e3f688c6a9141](https://github.com/TheNetAdmin/zjuthesis/assets/58238523/0aec7587-87ca-46d4-8fac-34463a5d85c9)

我想加入这一页到论文中，请问该如何操作？

**Comment by xfz329** [comment]
可按以下方式进行修改。
1 在\page\graduate文件夹下新建一个tex文件，如fund.tex，并在其中编辑相应文字内容。
2 修改\page\graduate\previous.tex文件，修改13行代码为
```tex
\inputpage{fund}
\inputpage{acknowledgement}
```
个人感觉这个说明在盲审时被隐去，所以插在这个位置

**Comment by xfz329** [comment]
才发现你不想加入目录，那上述方式不太完美，可以按下面方式进行
1 在\page\graduate文件夹下新建一个tex文件，如fund.tex，并在其中编辑相应文字内容。
2 修改\page\graduate\cover.tex文件为
```tex
\inputpage{cover-chn}
\inputpage{title-chn}
\inputpage{title-eng}
\inputpage{auth-claim}
\ifthenelse{\equal{\BlindReview}{true}}
{
}
{
    \inputpage{fund}
}
```

这样应该就可以了

**Reply by chengzhongyi8** [reply]
谢谢！
